pkgs/misc/plasma-hud/default.nix

```
@@ -0,0 +1,38 @@
+{lib, python3, fetchFromGitHub, rofi, gobject-introspection}:
+with python3.pkgs;
+buildPythonApplication rec{
```

```suggestion
{ lib, python3, fetchFromGitHub, rofi, gobject-introspection }:

buildPythonApplication rec{
```

Please apply the with where necessary.